ui/package.json

```diff
@@ -1,14 +1,29 @@
 {
   "private": true,
   "scripts": {
+    "copy-bootswatch-fonts": "yarn cpr node_modules/bootswatch/fonts/ src/main/resources/webapp/fonts/bootstrap -o",
```

Comment:
If you want, we can simplify `yarn cpr` into `cpr`, `yarn mkdirp` into `mkdirp`, and `yarn rimraf` into `rimraf`.